integrationtests/ui/instance_editor_test2.py

```diff
@@ -8,10 +8,9 @@
 #
 # Thanks for using Enthought open source!
 
-from traits.api import *
-from traitsui.api import *
-from traitsui.instance_choice \
-    import InstanceChoice, InstanceFactoryChoice
+from traits.api import HasStrictTraits, Instance, Int, List, Regex, Str
+from traitsui.api import InstanceEditor, Item, View
+from traitsui.instance_choice import InstanceFactoryChoice
```

Comment:
```suggestion
from traitsui.api import InstanceEditor, InstanceFactoryChoice, Item, View
```

Reply:
Right, I forgot we'd made them part of the API!